scripts/cv_api_master.py

```diff
@@ -1,127 +1,470 @@
+                    "path": "random_flip",
+                    "title": "Random Flip layer",
+                    "generate": ["keras_cv.layers.RandomFlip"],
+                },
+                {
+                    "path": "random_guassian_blur",
+                    "title": "Random Gaussian Blur layer",
+                    "generate": ["keras_cv.layers.RandomGaussianBlur"],
+                },
+                {
+                    "path": "random_hue",
+                    "title": "RandomHue layer",
+                    "generate": ["keras_cv.layers.RandomHue"],
+                },
+                {
+                    "path": "random_jpeg_quality",
+                    "title": "Random Jpeg Quality layer",
+                    "generate": ["keras_cv.layers.RandomJpegQuality"],
+                },
+                {
+                    "path": "random_rotation",
+                    "title": "Random Rotation layer",
+                    "generate": ["keras_cv.layers.RandomRotation"],
+                },
+                {
+                    "path": "random_saturation",
+                    "title": "RandomSaturation layer",
+                    "generate": ["keras_cv.layers.RandomSaturation"],
+                },
+                {
+                    "path": "random_sharpness",
+                    "title": "RandomSharpness layer",
+                    "generate": ["keras_cv.layers.RandomSharpness"],
+                },
+                {
+                    "path": "random_shear",
+                    "title": "RandomShear layer",
+                    "generate": ["keras_cv.layers.RandomShear"],
+                },
+                {
+                    "path": "random_translation",
+                    "title": "Random Translation layer",
+                    "generate": ["keras_cv.layers.RandomTranslation"],
+                },
+                {
+                    "path": "random_zoom",
+                    "title": "Random Zoom layer",
+                    "generate": ["keras_cv.layers.RandomZoom"],
+                },
+                {
+                    "path": "repeated_augmentation",
+                    "title": "Repeated Augmentation layer",
+                    "generate": ["keras_cv.layers.RepeatedAugmentation"],
+                },
+                {
+                    "path": "solarization",
+                    "title": "Solarization layer",
+                    "generate": ["keras_cv.layers.Solarization"],
+                },
+                {
+                    "path": "vectorized_image_augmentation",
+                    "title": "Vectorized BaseImage Augmentation layer",
+                    "generate": ["keras_cv.layers.VectorizedBaseImageAugmentationLayer"],
+                },
+            ],
         },
         {
-            "path": "equalization",
-            "title": "Equalization layer",
-            "generate": ["keras_cv.layers.Equalization"],
+            "path": "convolution/",
```

Comment:
Please create a separate category "modeling" and move the conv layers and the encoder layers there.